### PR TITLE
feat(serialization): offer option to omit stringification

### LINKED ISSUE #103: POSTing raw string value gets stringified
Is there a way to `POST` to an endpoint at the end of a traversal and not have it stringified? What I'm looking to do is POST payload of TSV data. Unfortunately, when I send `"data"` to the server, it ends up getting sent as `""data""` (notice the double quotes). It looks like I could potentially override jsonReplacer that was added in https://github.com/traverson/traverson/pull/69/files, but wanted to see if I was missing something a more appropriate way. Appreciate any guidance.

### PR DESCRIPTION
**Pull request by basti1302**
fixes #103

### CODE REVIEW

File: api.markdown [hunk 1]
```diff
@@ -143,6 +143,17 @@ Returns the request builder instance to allow for method chaining.
 
 <a name="builder-withRequestLibrary"></a>`withRequestLibrary(request)`: Injects a custom request library. Returns the request builder instance to allow for method chaining.
 
+<a name="builder-sendRawPayload"></a>`sendRawPayload(flag)`: With this option enabled, the payload of the last request at the end of the
+traversal will be send as is, without stringifying it. The default is false, which means that usually Traverson assumes the payload is passed as a JavScript object which will then be stringified (which is the right thing to do for JSON based MIME types like `application/json`. If you want to handle the serialization yourself and don't want Traverson to interfere, this option should be set to true.
```
Comment (travi) [inline api.markdown]: s/send/sent

File: api.markdown [hunk 2]
```diff
@@ -179,6 +190,8 @@ If the method is called without arguments (or the first argument is undefined or
 
 <a name="builder-getJsonParser"></a>`getJsonParser()`: Returns the custom JSON parser function set by `parseResponseBodiesWith` or the standard parser function, if a custom one has not been set.
 
+<a name="builder-sendsRawPayload"></a>`sendsRawPayload()`: Returns true if the payload will be send without stringifying it first.
```
Comment (travi) [inline api.markdown]: s/send/sent

File: lib/builder.js [hunk 3]
```diff
@@ -364,6 +365,29 @@ Builder.prototype.useAutoHeaders = function(flag) {
   return this;
 };
 
+/**
+ * With this option enabled, the payload of the last request at the end of the
+ * traversal will be send as is, without stringifying it. The default is false,
```
Comment (travi) [inline lib/builder.js]: s/send/sent

File: lib/builder.js [hunk 4]
```diff
@@ -489,6 +513,13 @@ Builder.prototype.setsAutoHeaders = function() {
   return this.autoHeaders;
 };
 
+/**
+ * Returns true if the given payload will be send without stringifying it first.
```
Comment (travi) [inline lib/builder.js]: s/send/sent

File: lib/builder.js [hunk 5]
```diff
@@ -364,6 +365,29 @@ Builder.prototype.useAutoHeaders = function(flag) {
...
+ * serialization yourself and don't want Traverson to interfere, this option
+ * should be set to true.
+ *
+ * If the method is called without arguments (or the first argument is undefined
+ * or null), this option is switched on, otherwise the argument is
+ * interpreted as a boolean flag. If it is a truthy value, the option is
+ * switched to on, if it is a falsy value (but not null or
+ * undefined), the option is switched off.
+ */
+Builder.prototype.sendRawPayload = function(flag) {
```
Comment (richardson-trevor) [inline lib/builder.js]: 👍

File: lib/builder.js (hunk 4 above)
Comment (basti1302) [reply lib/builder.js]: All borked past participles fixed, thanks for playing spell checker for me :-)

**Pull request merged**